### PR TITLE
Call `DBI::Id()` without names

### PR DESCRIPTION
**Pull request by krlmlr**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -37,6 +37,8 @@ Suggests:
 VignetteBuilder: 
     knitr
 ByteCompile: true
+Config/Needs/check:
+    r-dbi/DBI
```
Comment (jennybc) [inline DESCRIPTION:41]: Do we really need this long-term? I can see how, today, it is how we're getting the DBI dev candidate. But going forward `DBI (>= 1.2.0)` in `Suggests` is, I think, all we need.

**Review by jennybc — COMMENTED** [review]
I just re-ran the failed checks.

File: DESCRIPTION (hunk 1 above)
Comment (krlmlr) [reply DESCRIPTION]: Thanks. Fully agree.

DBI 1.2.0 is in CRAN's queue, it needs manual intervention but should come through tomorrow.

**Comment by jennybc** [comment]
This is promising. I'll wait until DBI hits CRAN to truly initiate a glue release, but you can certainly say that a fixed version of glue will be submitted to CRAN soon after DBI goes through. Maybe it can complete before the holiday CRAN pause or, if not, shortly after things re-open.

**Comment by krlmlr** [comment]
Thank you!

**Comment by krlmlr** [comment]
Apologies for crying wolf -- this PR requires DBI 1.2.0, but it turns out that the current glue works with DBI 1.2.0 too (after I fixed a glitch there). Still, better to have that in early.

**Pull request merged**